src/encoded/static/components/browse/components/file-tables.js

```diff
@@ -4,11 +4,238 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import _ from 'underscore';
 import memoize from 'memoize-one';
-import { FacetList } from './FacetList';
-import { StackedBlock, StackedBlockList, StackedBlockName, StackedBlockTable, FileEntryBlock, FilePairBlock, FileEntryBlockPairColumn } from './StackedBlockTable';
-import { expFxn, Filters, console, isServerSide, analytics, object, Schemas, typedefs } from './../../util';
+import url from 'url';
+import { Button, DropdownButton, MenuItem } from 'react-bootstrap';
+import { StackedBlock, StackedBlockList, StackedBlockName, StackedBlockNameLabel, StackedBlockTable, FileEntryBlock, FilePairBlock } from './StackedBlockTable';
+import { expFxn, console, isServerSide, analytics, object, Schemas, typedefs, fileUtil, navigate } from './../../util';
 
-var { Item } = typedefs;
+var { Item, ExperimentSet } = typedefs;
+
+
+
+
+class FileColumnActionsBtn extends React.PureComponent {
+
+    static hostFromHref = memoize(function(href){
+        const hrefParts = (href && url.parse(href)) || null;
+        const host = hrefParts && (
+            (hrefParts.protocol || '') +
+            (hrefParts.hostname ? '//' +  hrefParts.hostname + (hrefParts.port ? ':' + hrefParts.port : '') : '')
+        );
+        return host || null;
+    });
+
+    static epiGenomeAssemblyMapping = {
+        'GRCh38' : 'hg38',
+        'GRCm38' : 'mm10'
+    };
+
+    constructor(props){
+        super(props);
+        this.isFileHIC = this.isFileHIC.bind(this);
+        this.higlassButton = this.higlassButton.bind(this);
+        this.juiceboxButton = this.juiceboxButton.bind(this);
+        this.epigenomeButton = this.epigenomeButton.bind(this);
+    }
+
+    isFileHIC(){
+        const file = this.props.file;
+        const fileFormat = fileUtil.getFileFormatStr(file);
+        return (file && file.href && (
+            // Needs an href + either it needs a file format of 'hic' OR it has a detailed file type that contains 'hic'
+            (fileFormat && fileFormat === 'hic')
+            || (file.file_type_detailed && file.file_type_detailed.indexOf('(hic)') > -1)
+        ));
+    }
+
+    higlassButton(){
```

Comment:
This method is defined but actually called/used at the moment.
Should be refactored from button into a separate indicator as before (?).